### PR TITLE
Don't check excluder versions when they're not enabled

### PR DESCRIPTION
**Pull request by sdodson**
Fixes https://github.com/openshift/origin/issues/13846 on master

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

File: playbooks/common/openshift-cluster/upgrades/disable_excluder.yml [hunk 1]
```diff
@@ -5,12 +5,13 @@
...
   - include: pre/validate_excluder.yml
     vars:
-      #repoquery_cmd: repoquery_cmd
-      #openshift_upgrade_target: openshift_upgrade_target
-      excluder: "{{ item }}"
-    with_items:
-    - "{{ openshift.common.service_type }}-docker-excluder"
-    - "{{ openshift.common.service_type }}-excluder"
+      excluder: "{{ openshift.common.service_type }}-docker-excluder"
+    when: enable_docker_excluder | default(enable_excluders) | default(True) | bool
```
Comment (ingvagabund) [inline playbooks/common/openshift-cluster/upgrades/disable_excluder.yml]: This duplicates the information about how the enable_docker_excluder and enable_openshift_excluder are set. However, we can fix it later.

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by sdodson** [comment]
This is failing because registry.ops.openshift.com/openshift3/ose:v3.5 is v3.5.5.5 not v3.5.5.8 which is in the yum repo.

```
# docker run --entrypoint rpm registry.ops.openshift.com/openshift3/ose:v3.5 -q atomic-openshift
atomic-openshift-3.5.5.5-1.git.0.f2e87ab.el7.x86_64
```

**Pull request merged**